src/conformal_cubed_sphere.jl

```diff
@@ -57,4 +57,48 @@ function conformal_cubed_sphere_mapping(x, y)
     return X, Y, Z
 end
 
+"""
+   lamrofnoc_cubed_sphere_mapping(X, Y, Z)
+
+Inverse mapping for conformal cube sphere for quadrant of north-pole face in which X and Y are both
+positive. All other mappings to other cube face coordinates can be recovered from rotations of this map.
+There a 3 other quadrants for the north-pole face and five other faces for a total of twenty-four quadrants.
+Because of symmetry only the reverse for a single quadrant is needed. Because of branch cuts and the complex
+transform the inverse mappings are multi-valued in general, using a single quadrant case allows a simple
+set of rules to be applied.
+
+"""
+function lamrofnoc_cubed_sphere_mapping(X, Y, Z)
```

Comment:
Took a while to figure out what `lamrofnoc` meant haha.

Should we call the function `conformal_cubed_sphere_inverse_mapping` to match up with the existing name?